app/src/main/res/xml/pref_main.xml

```diff
@@ -8,16 +8,44 @@
         android:icon="?attr/save_icon_color"
         android:key="settings_save_last_position"
         android:summary="@string/settings_summary_save_position"
-        android:title="@string/settings_title_save_position"/>
+        android:title="@string/settings_title_save_position" />
     <SwitchPreference
         android:icon="?attr/fixed_ui_icon_color"
         android:key="disable_fixed_ui"
-        android:title="@string/pref_disable_fixed_ui"
-        android:summary="@string/pref_disable_fixed_ui_summary"/>
+        android:summary="@string/pref_disable_fixed_ui_summary"
+        android:title="@string/pref_disable_fixed_ui" />
     <SwitchPreference
         android:icon="?attr/bounce_icon_color"
         android:key="enable_bounce_effect"
         android:summary="@string/pref_enable_bounce_effect_summary"
         android:title="@string/pref_enable_bounce_effect"
         android:visibility="gone" />
+
+    <PreferenceCategory android:title="@string/floating_dialog_settings">
+        <SwitchPreference
```

Comment:
Better to use MultiSelectListPreference for those as takes less space and we need to make sure that at least one bar is selected, otherwise i'd be pointless.